### DISCUSSION TITLE
When can we expect a release that contains default Queue Types for Vhosts

### DISCUSSION THREAD

**Discussion by jbexten** [michaelklishin/rabbit-hole#266] [General]
We are in the midst of moving our rabbitMQ servers to Kubernetes and we are using the messaging topology operator. We would like to take advantage of this opportunity of changing servers to also upgrade our queues to quorum queues. However, we would like to use DefaultQueueType on the new vhosts so that we can use federation with our old servers during the migration without having to specify a parameter on the queues. 

I see that DefaultQueueType has already been added, however there hasn't been a release since February. When can we expect a release so that the messaging-topology-operator can pull in this new functionality?

Thank You

**Comment by michaelklishin** [comment]
[`2.14.0` is out](https://github.com/michaelklishin/rabbit-hole/releases/tag/v2.14.0).
